web/src/components/vis/VisTile.vue

```diff
@@ -11,28 +11,61 @@ export default {
     },
   },
 
+  data() {
+    return {
+      scale: 1,
+    };
+  },
+
   computed: {
-    hasControls() {
```

Comment:
It doesn't seem like this should have been deleted--it's meant to control whether the menu icon appears in the toolbar.

Reply:
no it controlled whether the overall toolbar on top was rendered. The menu icon is rendered in the Plot tiles directly.